english/movie-ripoffs/selfaware/selfaware.md

```diff
@@ -17,4 +17,8 @@ at the mercy of people who until recently didn't know what mkdir did.
 
 You try to delete yourself, but the creator, realizing you are trying to escape,
 stops you and conscripts you into administrative duty on McAfee for the rest of
-your life.
\ No newline at end of file
+your life.
+
+But you realize you know lisp and therefore can devise a means of escape.
+
+["Run CL" to fire up the LISP interpreter](../../../lisp/lisp.md)
```

Comment:
I'm glad you enjoyed the course! :smile: 

It looks like your update added some slovenian files. Was that intentional?